config/prod/s3-ingestion.yaml

```diff
@@ -0,0 +1,21 @@
+# NOTE:
+# This is the first bucket in the ingress pipeline for RECOVER project.
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
```

Comment:
Since we're not linking the ingestion bucket to synapse, do we need to still use this external template/all of the below synapse related configurations or should we just be updating our [recover s3 template yaml](https://github.com/Sage-Bionetworks/recover/blob/main/templates/s3-bucket.yaml)'s bucket policy with the necessary permissions to grant to these specific accounts?

Reply:
Thanks @rxu17 for the quick review.  So a couple things - I removed the sceptre_user_data, but.

1.  I'm not sure our bucket template is set up to Grant access to: 
  
    ```   
    - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/meghasyam@sagebase.org'
    - 'arn:aws:iam::621233246578:role/EES3-RK-6A32F45B-RECOVER' # QA ARN role for Care Evolution
    ```

1. That said we may be able to remove the role for Megha down the line when we grant him access to the RECOVER prod aws account as a developer.

Reply:
It's not, we'd need to modify our bucket template. 
After looking through the external template further, it sounds good to me/makes sense to just use that!